ventor_base/__init__.py

```diff
@@ -11,6 +11,14 @@ def _post_init_hook(cr, registry):
     """
     env = api.Environment(cr, SUPERUSER_ID, {})
 
+    all_stock_inventory = env['stock.inventory'].search([
+            ('state', '=', 'draft'),
+            ])
```

Comment:
we need to also update non-draft inventories to hide them

Reply:
Done